packages/sdk/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -301,7 +301,9 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
     } else if (isValidAddress(target)) {
       return target
     } else {
-      throw new Error(`${target} is neither a StableToken or a valid Address`)
+      throw new Error(
+        `${target} is neither CeloContract.StableToken, CeloContract.StableTokenEUR or a valid Address`
```

Comment:
nit: would it make sense here to generalize this to something like `${target} is not StableTokenContract or a valid Address`?